### PR TITLE
Sw 4853 display deputy team details

### PR DESCRIPTION
**Pull request by kate-49**

### CODE REVIEW

File: web/assets/main.scss [hunk 1]
```diff
@@ -1,10 +1,11 @@
 $govuk-include-default-font-face: false;
 $govuk-moj-table-background-color: #e7f2fb;
-$moj-assets-path: "/supervision/deputy-hub/assets/";
+$moj-assets-path: "/supervision/deputies/public-authority/assets/";
+$govuk-assets-path: "/supervision/deputies/public-authority/assets/";
```
Comment (denialanderror) [inline web/assets/main.scss:4]: Is there a reason why these have the same path? I don't know the difference between these tbh.

File: web/assets/scss/pa-hub.scss [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+    span.rem_warning {
+      background: url("/public/images/flag.svg") no-repeat 3px 3px;
+      padding-left: 20px;
+      font-weight: bold;
+      color: #d4351c;
+      font-size: 16px;
+      font-size: 1rem;
+      // background: #d4351c;
+      // display: inline-block;
+      // padding: 0 5px;
```
Comment (denialanderror) [inline web/assets/scss/pa-hub.scss:140]: If this commented out code is not needed, remove it

**Review by denialanderror — DISMISSED** [review]
One question for my benefit and some commented out code to remove but approved

File: web/assets/main.scss (hunk 1 above)
Comment (hiboabd) [reply web/assets/main.scss:4]: There is a different asset path variable name for each front end repo. The `$moj-assets-path` variable is for the `@ministryofjustice/frontend `repo and the `$govuk-assets-path` is for the `govuk-frontend` repo. Both need to be set to have the right url to locate the images

**Review by denialanderror — DISMISSED** [review]

**Review by denialanderror — APPROVED** [review]

**Pull request merged**